feat(surrealdb): add updateStrategy (#20)
surrealdb/helm-charts: charts/surrealdb/README.md

| replicaCount | int | `1` | The number of SurrealDB pods to run  Note that you usually scale this only when the backend supports it. For example, if you specify volumes and volumeMounts to make this SurrealDB instance stateful, you should not scale it, as it will result in two or more instances writing to the same volume or working independently. |
| resources | object | `{}` | Resource limits and requests |
| securityContext | object | `{}` (See [values.yaml]) | SurrealDB container-level security context |

> | strategy | object | `{"type":"RollingUpdate"}` | Only really applicable when the deployment has an RWO PV attached Since replicas = 1, an update can get "stuck", as the previous pod remains attached to the PV, and the "incoming" pod can never start. Changing the strategy to "Recreate" will terminate the single previous pod, so that the new, incoming pod can attach to the PV |

Reviewer: ```suggestion
| strategy | object | `{"type":"RollingUpdate"}` | Set to `{"type":"Recreate"}` when the deployment has an RWO PV attached and replicas = 1. Otherwise, an update can get stuck, as the previous pod remains attached to the PV, and the "incoming" pod can never start. Changing the strategy to "Recreate" will terminate the single previous pod, so that the new, incoming pod can attach to the PV |
```

I think this would be a bit clearer?
